### PR TITLE
Change authorship order

### PR DESCRIPTION
**Pull request by flaviorangel**

### CODE REVIEW

File: paper/paper.md [hunk 1]
```diff
@@ -4,20 +4,28 @@ title: 'ROSS - Rotordynamic Open Source Software'
...
    affiliation: 2
  - name: Flavio Rangel
+   orcid: 0000-0002-4852-8141
    affiliation: 2
- - name: Gabriel Bachman
+ - name: Juliana Vianna
+   orcid: 0000-0000-0000-0000
+   affiliation: 2
+ - name: Thiago Ritto
+   orcid: 0000-0000-0000-0000
```
Comment (rodrigomoliveira1) [inline paper/paper.md]: It's not exactly a change request. It's only to remember the members to send their orcid numbers to add in here.

Comment (raphaeltimbo) [reply paper/paper.md]: Here is my number: 0000-0001-7689-5486

Comment (rodrigomoliveira1) [reply paper/paper.md]: Here is mine: 0000-0002-9996-6600

Comment (JuliaMota) [reply paper/paper.md]: My number is 0000-0002-9547-9629

Comment (GabrielBachmann) [reply paper/paper.md]: My number is 0000-0003-4401-8408

**Pull request merged**